Tiltfile

```diff
@@ -48,5 +48,13 @@ local_resource(
     cmd='helm install openshift-vertical-pod-autoscaler -n openshift-vertical-pod-autoscaler oci://ghcr.io/stakater/charts/openshift-vertical-pod-autoscaler'
     )
 
+# Wait until VPA CRD becomes available
+local_resource(
+    'wait-for-crds', 
+    cmd='timeout 300s bash -c "until kubectl wait --for condition=Established crd/verticalpodautoscalers.autoscaling.k8s.io; do sleep 10; done"',
+    resource_deps=[
+        'openshift-vertical-pod-autoscaler'
+    ])
```

Comment:
Would it make sense to kind of merge these two commands together? Just a small suggestion, in case it makes semantic sense. I'm not well versed in Tilt, so this might be silly.

Reply:
@d3adb5 can you plz create separate ticket for this so, we can merge this one?

Reply:
I don't see a reason to create an issue for what I think is a style choice, so I'll just resolve the thread for now and leave my approval. :+1: